src/components/MatchCriterion/view/IncomingSectionStatic/IncomingSectionStatic.test.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+import '../../../../../test/jest/__mock__';
+import { translationsProperties } from '../../../../../test/jest/helpers';
+
+import { IncomingSectionStatic } from './IncomingSectionStatic';
+
+const incomingSectionStaticWithText = {
+  staticValueDetails: {
+    staticValueType: 'TEXT',
+    text: 'test text',
+  },
+};
+const incomingSectionStaticWithoutText = {
+  staticValueDetails: {
+    staticValueType: 'TEXT',
+    text: '',
+  },
+};
+const incomingSectionStaticWithNumber = {
+  staticValueDetails: {
+    staticValueType: 'NUMBER',
+    number: 'test text',
+  },
+};
+const incomingSectionStaticWithoutNumber = {
+  staticValueDetails: {
+    staticValueType: 'NUMBER',
+    number: '',
+  },
+};
+const incomingSectionStaticWithDate = {
+  staticValueDetails: {
+    staticValueType: 'EXACT_DATE',
+    exactDate: '12/31/2010',
+  },
+};
+const incomingSectionStaticWithoutDate = {
+  staticValueDetails: {
+    staticValueType: 'EXACT_DATE',
+    exactDate: '',
+  },
+};
+const incomingSectionStaticWithDateRange = {
+  staticValueDetails: {
+    staticValueType: 'DATE_RANGE',
+    fromDate: '12/31/2010',
+    toDate: '12/31/2011',
+  },
+};
+const incomingSectionStaticWithoutDateRange = {
+  staticValueDetails: {
+    staticValueType: 'DATE_RANGE',
+    fromDate: '',
+    toDate: '',
+  },
+};
+const emptyIncomingSectionStatic = { staticValueDetails: null };
+
+const renderIncomingSectionStatic = ({ staticValueDetails }) => {
+  const component = (
+    <IncomingSectionStatic
+      staticValueDetails={staticValueDetails}
+    />
+  );
+
+  return renderWithIntl(component, translationsProperties);
+};
+
+describe('IncomingSectionStatic view', () => {
+  it('should be rendered empty', () => {
+    const { queryByText } = renderIncomingSectionStatic(emptyIncomingSectionStatic);
+
+    expect(queryByText('test text')).toBeNull();
```

Comment:
Can we do smth like this?
```suggestion
    expect(valueElement).toBeNotDefined();
```